continuous_integration/azure/install.sh

```diff
@@ -62,9 +62,13 @@ elif [ "$DISTRIB" == "nightly" ]; then
 elif [ "$DISTRIB" == "minimal" ]; then
     python -m pip install --upgrade . pytest pytest-cov coverage
 elif [ "$DISTRIB" == "ubuntu" ]; then
-    sudo apt-get install --fix-missing python3-numpy python3-matplotlib python3-pip python3-coverage optipng graphviz
-    python3 -m pip install --upgrade pip setuptools
-    python3 -m pip install -r dev-requirements.txt | cat
+    sudo apt-get install --fix-missing python3-numpy python3-matplotlib python3-pip python3-coverage optipng graphviz \
+      libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0 libopengl0 libegl1 libosmesa6 mesa-utils libxcb-shape0
+    python3 -m pip install --upgrade pip setuptools wheel
+    python3 -m pip install -r dev-requirements.txt
+    python3 -m pip install "vtk<9.2" pyqt5
```

Comment:
Stupid question - don't we install these above in `azure-pipelines.yml` here: https://github.com/sphinx-gallery/sphinx-gallery/pull/1046/files#diff-7915b9b726a397ae7ba6af7b9703633d21c031ebf21682f3ee7e6a4ec52837a5R62

Reply:
Not sure. At some point a bigger refactoring of CIs to unify installations would be nice. But not in this PR :)

Reply:
... oh actually, no we don't install them there. That is for the Windows job. These lines are used by the Linux job